.github/workflows/ci.yaml

```diff
@@ -1,68 +1,71 @@
-name: CI
+name: "CI"
 
 on:
     push:
-        branches: [ main ]
-    pull_request:
+        branches:
+            - "main"
+    pull_request: null
 
 jobs:
     tests:
-        runs-on: 'ubuntu-latest'
+        runs-on: "ubuntu-latest"
         name: "Test ${{ matrix.php }}"
         strategy:
             fail-fast: true
             matrix:
-                php: [ '8.1' ]
+                php:
+                    - "8.1"
                 include:
-                    - php: '7.4'
-                      skip_static_checks: 'true'
-                    - php: '8.0'
-                      skip_static_checks: 'true'
+                    - php: "7.4"
+                      skip_static_checks: true
+                    - php: "8.0"
+                      skip_static_checks: true
 
         steps:
-            -   name: 'Check out repository code'
-                uses: actions/checkout@v2
+            -   name: "Check out repository code"
+                uses: "actions/checkout@v2"
 
-            -   name: Setup PHP
-                uses: shivammathur/setup-php@v2
+            -   name: "Setup PHP"
+                uses: "shivammathur/setup-php@v2"
                 with:
-                    php-version: ${{ matrix.php }}
-                    tools: composer
-                    coverage: pcov
+                    php-version: "${{ matrix.php }}"
+                    tools: "composer"
+                    coverage: "pcov"
 
-            -   name: Install Composer dependencies
-                uses: ramsey/composer-install@v2
+            -   name: "Install Composer dependencies"
+                uses: "ramsey/composer-install@v2"
 
-            -   name: Install CoversValidator Composer dependencies
-                if: matrix.skip_static_checks != 'true'
-                uses: ramsey/composer-install@v2
+            -   name: "Install CoversValidator Composer dependencies"
+                if: "! matrix.skip_static_checks"
```

Comment:
I think you need to keep the `!= 'true'` there: we are not checking real booleans but against a `'true'` string

Reply:
That true string is gone.
We now have a real YAML true!

Reply:
... and work as expected:

![kép](https://user-images.githubusercontent.com/952007/174492055-93dd2886-e8eb-4041-b641-1f113d1e2e80.png)

steps are being skipped 🥳

Reply:
Got it.
The first "Mark ..." step needs no `!`